test/testHelper.js

```diff
@@ -63,12 +62,6 @@ async function getDb (database) {
   return db
 }
 
-async function createPgCrypto (database) {
-  const db = await getDb(database)
-  await db.executeSql('create extension if not exists pgcrypto')
-  await db.close()
-}
-
```

Comment:
Removing this as tests fail; PG images already have pgcrypto installed, but for some reason `if not exists` doesn't really work in this suite